### PR TITLE
Add  LICENSE info to README.md

### LINKED ISSUE #3404: LICENSE info missing from README.md
## What you expected to happen?
LICENSE info should be present in the README.md

## What happened?
LICENSE info missing from README.md

## How to reproduce it?
NA

## Anything else we need to know?
NA

## Versions:

```
$ scope version
$ docker version
$ uname -a
$ kubectl version
```

## Logs:
```
$ docker logs weavescope
```
or, if using Kubernetes:
```
$ kubectl logs <weave-scope-pod> -n <namespace> 
```

### PR DESCRIPTION
**Pull request by akshatnitd**
Fixes #3404 
Add  LICENSE info to README.md

### CODE REVIEW

**Comment by dholbach** [comment]
Is it worth pointing to VENDORED_CODE.md as well?

**Comment by akshatnitd** [comment]
@dholbach Should I make a separate section for it?

**Comment by dholbach** [comment]
Maybe just add a sentence saying "Find more details about the licenses of vendored code <here>."

**Comment by akshatnitd** [comment]
@dholbach Made the changes :)

**Comment by rndstr** [comment]
sorry to be pedantic but i would prefer if we didn't use ['here' links](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/)

**Comment by dholbach** [comment]
@akshatnitd ^ Do you think you can take a look at this?

**Comment by akshatnitd** [comment]
@dholbach Sorry for the delay. Made the change :v:

File: README.md [hunk 1]
```diff
@@ -59,3 +59,7 @@ If you have any questions, feedback, or problems with Scope:
 - [File an issue](https://github.com/weaveworks/scope/issues/new).
 
 Your feedback is always welcome!
+
+### License
+Scope is licensed under the Apache License, Version 2.0. See [LICENSE](LICENSE) for the full license text. <br />
```
Comment (dholbach) [inline README.md]:
```suggestion
Scope is licensed under the Apache License, Version 2.0. See [LICENSE](LICENSE) for the full license text.  
```

You can use a double space at the end of the line to introduce a <br />

**Comment by akshatnitd** [comment]
@dholbach Made the changes :v:

**Pull request merged**